metadata/base/service.yaml

```diff
@@ -0,0 +1,12 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: metadata-service
+spec:
+  selector:
+    deployment: metadata
+  type: LoadBalancer
```

Comment:
Is it possible to make this NodePort? It might be better for non-cloud clusters.

Reply:
This is an initial pr to set up the kustomize files. I set it as "LoadBalancer" as other components. We can add overlay to customize it later if it is needed.